packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -2,16 +2,21 @@ import _ from 'lodash';
 import path from 'path';
 import url from 'url';
 import logger from './logger';
-import {tempDir, fs, util, zip, net, timing, node} from '@appium/support';
+import {tempDir, fs, util, zip, timing, node} from '@appium/support';
 import LRU from 'lru-cache';
 import AsyncLock from 'async-lock';
 import axios from 'axios';
+import B from 'bluebird';
 
 const IPA_EXT = '.ipa';
 const ZIP_EXTS = ['.zip', IPA_EXT];
 const ZIP_MIME_TYPES = ['application/zip', 'application/x-zip-compressed', 'multipart/x-zip'];
 const CACHED_APPS_MAX_AGE = 1000 * 60 * 60 * 24; // ms
 const MAX_CACHED_APPS = 1024;
+const HTTP_STATUS_NOT_MODIFIED = 304;
+const DEFAULT_REQ_HEADERS = Object.freeze({
+  'user-agent': 'Appium',
```

Comment:
guess it doesn't really matter but might want to stuff a version in there too.

Reply:
do we have some helper, which allows to fetch the version number?

Reply:
could just use basedriver's version number so you don't have to look outside this package

```
Appium (BaseDriver vX.Y.Z)
```

Reply:
probably an example of getting the version of base driver package?

Reply:
something like `require('../path/to/package.json').version`

Reply:
added